src/Builder/FormContractor.php

```diff
@@ -111,7 +116,7 @@ public function getDefaultOptions($type, FieldDescriptionInterface $fieldDescrip
                 $options['model_manager'] = $fieldDescription->getAdmin()->getModelManager();
 
                 break;
-            case 'Sonata\AdminBundle\Form\Type\Modeltype':
+            case 'Sonata\AdminBundle\Form\Type\ModelType':
```

Comment:
how to typo :-/
php 5.6 with the ::class constant will help us to avoid this kind of error in the future!